.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: 'Build'
+on:
+  push:
+    branches:
+      - master
```

Comment:
question: with Jenkins, we used to build all branches but with this new system, I wonder if we should just build for PRs since the built status is only useful to block/allow PRs and the built artefacts are only published for PRs.

Reply:
OK :+1: